doc/reference/std/errors.md

```diff
@@ -426,7 +426,23 @@ displaying the exception with `display-exception`).
 ```
 
 Invokes `thunk` with an exception handler that dumps the exception
-stack trace with `dump-stack-trace!`.
+stack trace with `dump-stack-trace!`
+if `(dump-stack-trace?)` is true (the default).
+
+### dump-stack-trace?
```

Comment:
I don't like the existence of this, but I can be convinced otherwise.

Reply:
In tests, in UIs, etc., I want to be able to show the message without the backtrace. Even when I do show the backtrace, I want to repeat the non-backtrace message *after* the backtrace.